queue.go

```diff
@@ -103,7 +103,12 @@ func (f ProcessorFunc) Process(ctx context.Context, rs ...Record) error {
 // Topic represents a destination topic where to produce a message/record.
 type Topic string
 
+// TopicConsumer is used to monitor a set of consumer topics.
 type TopicConsumer struct {
-	Topic    Topic
+	// Optional topic to monitor.
+	Topic Topic
```

Comment:
Do we want to keep this, or could we (eventually) just have the regex and make it required? It appears that Google Cloud Monitoring also supports regex matches: https://cloud.google.com/monitoring/api/v3/filters#filter_syntax

Reply:
I expect we'll remove it later once we have no use for topics with known names. Until then, we need to keep it to maintain the current creation and monitoring mechanics.